ui/admin/tests/acceptance/roles/read-test.js

```diff
@@ -82,4 +82,15 @@ module('Acceptance | roles | read', function (hooks) {
     await visit(urls.roles);
     assert.notOk(find('main tbody .hds-table__tr:nth-child(1) a'));
   });
+
+  test('can navigate to a role and it has correct grant scope', async function (assert) {
+    assert.expect(1);
+    const { name: scopeName } = this.server.schema.scopes.findBy({
+      id: instances.role.grant_scope_id,
+    });
+    await visit(urls.roles);
+    await click('main tbody .hds-table__tr:nth-child(1) a');
+    await a11yAudit();
+    assert.dom('.rose-dropdown [title]').hasText(scopeName);
```

Comment:
I think the goal of this test should be to confirm that we don't retrieve project scopes. I think this test would still pass even with the previous bug?

Reply:
@ZedLi good point

Reply:
+1 to Zed's comment 😉

Reply:
@ZedLi @calcaide updated the test.  Take a look and tell me what you think